modules/installation-dns-upi.adoc

```diff
@@ -46,7 +46,7 @@ where `n` is the number of control plane machines in the cluster. The DNS
 record must resolve to an unicast IPV4 address for the control plane machine,
 and the records must be resolvable from all the nodes in the cluster.
 
-|`etcd-server-ssl._tcp.<cluster_name>.<base_domain>`
+|`_etcd-server-ssl._tcp.<cluster_name>.<base_domain>`
```

Comment:
@jiajliu, the bug made it sound like only this record needed the _. Will you please confirm?